microsoftgraph/microsoft-graph-explorer-v4
# Fix: Update the cached queries in queries.ts

## Overview

- Closes #2036 
- This PR updates the set of cached queries in queries.ts which are essential when DevX API fails to return the sample queries.

## Testing Instructions

###  How to test this PR
* Checkout to fix/update-cached-queries
* Configure your browser to slow network or low connectivity
* Load graph explorer and observe fix
